src/src/com/microsoft/aad/adal/BrokerProxy.java

```diff
@@ -299,9 +300,12 @@ private AuthenticationResult getResultFromBrokerResponse(Bundle bundleResult) {
 
             // IDtoken is not present in the current broker user model
             UserInfo userinfo = UserInfo.getUserInfoFromBrokerResult(bundleResult);
+            final String tenantId = bundleResult.getString(AuthenticationConstants.Broker.ACCOUNT_USERINFO_TENANTID, "");
+            final Date expires = new Date(bundleResult.getLong(AuthenticationConstants.Broker.ACCOUNT_EXPIREDATE));
```

Comment:
this will mark the token as expired on Jan 1, 1970. if date is not returned from the broker then we should assume 1 hour as the default expiration time. This is how iOS and .NET work when date is not returned from the SERVER. In this case you are speaking to the broker that does return the value, but we should still bullet proof the code.